stable/cachet/README.md

```diff
@@ -1,6 +1,8 @@
 # cachet
 
-![Version: 1.2.7](https://img.shields.io/badge/Version-1.2.7-informational?style=flat-square) ![AppVersion: 2.3.15](https://img.shields.io/badge/AppVersion-2.3.15-informational?style=flat-square)
+
+
+![Version: 1.2.8](https://img.shields.io/badge/Version-1.2.8-informational?style=flat-square) ![AppVersion: 2.3.15](https://img.shields.io/badge/AppVersion-2.3.15-informational?style=flat-square) 
```

Comment:
Sorry but the CI tests are quite strict, these extra blank lines will cause both markdown lint and docs to fail 🙂

```suggestion
![Version: 1.2.8](https://img.shields.io/badge/Version-1.2.8-informational?style=flat-square) ![AppVersion: 2.3.15](https://img.shields.io/badge/AppVersion-2.3.15-informational?style=flat-square) 
```

Reply:
This is interesting, I did not manually touch those files, those additional line breaks must have been added during the template rendering.

I will investigate how they ended up there in the output of the gotmpl render and maybe PR a fix for that too if it's a bug.